flexget/plugins/clients/qbittorrent.py

```diff
@@ -134,9 +140,9 @@ def add_entries(self, task, config):
                     log.debug('temp: %s', ', '.join(os.listdir(tmp_path)))
                     entry.fail("Downloaded temp file '%s' doesn't exist!?" % entry['file'])
                     continue
-                self.add_torrent_file(entry['file'], form_data)
+                self.add_torrent_file(entry['file'], form_data, config)
```

Comment:
Don't pass the whole config if you just need it for the verify cert, pass just that